libmexclass/matlab/+libmexclass/+proxy/Proxy.m

```diff
@@ -72,4 +72,17 @@ function delete(obj)
             n = listLength(obj.AddedFields,indexOp,indexContext);
         end
     end
+
+    methods(Static, Access = public)
+        function proxy = create(name, arg)
+            arguments
+                name(1, 1) string {mustBeNonmissing}
+            end
+            arguments(Repeating)
+                arg
+            end
+            id = libmexclass.proxy.gateway("Create", name, arg);
+            proxy = libmexclass.proxy.Proxy(name=name, ID=id);
```

Comment:
I think this a typo? I believe this should be `Name` (first letter capitalized)?

Reply:
Good catch!